Makefile

```diff
@@ -2,7 +2,11 @@ LICENSE_DIR=./licenses/
 BUILD_DIR=./build
 DOCKER_DIR=./docker/
 SHELL := /bin/bash
-GO_VERSION=`cat GO_VERSION`
+ifndef GOTOOLCHAIN
```

Comment:
@eternal-flame-AD just to be sure: this should be `ifdef` instead of `ifndef` right? We only want to use the GOTOOLCHAIN variable if it's set otherwise fallback to the go.mod toolchain version. I'll change this on master.

Reply:
Oops, slipped through, yes it does seem like a typo